index.css

```diff
@@ -0,0 +1,154 @@
+*{
+    margin: 0;
+    padding: 0;
+    scroll-behavior: smooth;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    outline: none; border: none;
+    text-decoration: none;
+}
+
+html{
+    font-size: 60%;
+    scroll-behavior: smooth;
+    scroll-padding-top: 6rem;
+    overflow-x: hidden;
+}
+
+.header{
+    position: sticky;
+    top: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background-color: white;
+}
+
+.header .header__left .fa-bars{
+    font-size: 2rem;
+    color: #555;
+    border-radius: .5rem;
+    padding: 1.5rem 1.5rem;
+    cursor: pointer;
+    border: 1rem  rgba(18, 18, 18, 0.2);
+}
+
+.header #toggler{
+    display: none;
+}
+
+.header__left{
+    display: flex;
+     flex: 0.1;
+}
+
+.header__left fa-bars{
+    padding:4px;
+}
+
+.header__left img{
+    height: 5rem;
+    width: 8rem;
+}
+
+.header__middle{
+    display: flex;
+    flex: 0.7;
+    padding: 1rem;
+    margin-left: 15rem;
+}
+
+.header__middle form {
+    display: flex;
+    border: 1.5px solid #1211112c;
+    padding: 0.5rem 1rem;
+    border-radius: 25px;
+    font-size: 1.2rem;
+    width:60%;
+    cursor: pointer;
+    min-width: 20rem;
+    margin-left: 4rem;
+}
+
+.header__middle form input{
+    width: 90%;
+}
+
+.header__middle button{
+    display: flex;
+    cursor: pointer;
+    margin-left: 2rem;
+}
+
+.header__middle .fa-microphone{
+    display: flex;
+    font-size: 1.5rem;
+    padding-top: 1rem;
+    padding-left: 1rem;
+}
+
+.header__middle .fa-search{
+    display: flex;
+    font-size: 1.6rem;
+    padding-top: .3rem;
+    margin-left: 5rem;
+}
+
+.header__right{
+    display: flex;
+    flex: 0.2;
+    justify-content: space-around;
+    font-size: 1.5rem;
+}
+
+.header__right i{
+    font-size: 1.5rem;
+}
+
+main{
+    display: flex;
+    width: 80%;
+    margin: auto;
+}
+
+.main-video{
+    display: flex;
+    margin-top: 2%;
+}
+
+.sidebar{
+    display: flex;
+    flex-direction: column;
+    color: rgb(82, 80, 80);
+    padding: 5px !important;
+    cursor: pointer;
+    margin-top: 1%;
+}
+
+.content-video{
+    width: 30%;
+    margin-left: 10px;
+}
+
+.content-image{
+    position: relative;
+    width: 28rem;
+    cursor: pointer;
+    padding: 10px;
+}
```

Comment:
It would be better to add a single space between your styles to make code more clean.